docs/en/observability/apm/tab-widgets/install-agents.asciidoc

```diff
@@ -364,41 +349,45 @@ var apm = require('elastic-apm-node').start({
 
 *Install the agent*
 
-Install the PHP agent using one of the https://github.com/elastic/apm-agent-php/releases[published packages].
+Install the agent using one of the https://github.com/elastic/apm-agent-php/releases/latest[packages for supported platforms].
 
 To use the RPM Package (RHEL/CentOS and Fedora):
 
-[source,php]
+[source,bash]
 ----
 rpm -ivh <package-file>.rpm
 ----
 
 To use the DEB package (Debian and Ubuntu):
 
-[source,php]
+[source,bash]
 ----
 dpkg -i <package-file>.deb
 ----
 
 To use the APK package (Alpine):
 
-[source,php]
+[source,bash]
 ----
 apk add --allow-untrusted <package-file>.apk
 ----
 
-If you can’t find your distribution,
-you can install the agent by {apm-php-ref-v}/setup.html[building it from the source].
+If you can’t find your distribution, you can install the agent by building it from the source.
+The following instructions will build the APM agent using the same docker environment that Elastic uses to build our official packages.
+
+NOTE: The agent is currently only available for Linux operating system.
+
+1. Download the agent source from https://github.com/elastic/apm-agent-php/.
+2. Execute the following commands to build the agent and install it:
```

Comment:
Is there supposed to be a code block immediately following this line? Or should lines 383-385 be removed?

Reply:
Fixed in b4f3240